sim/core/spell.go

```diff
@@ -710,15 +693,15 @@ func (spell *Spell) ApplyAdditivePeriodicDamageBonus(percent int64) {
 }
 
 func (spell *Spell) updateBaseDamageMultiplier() {
-	spell.baseDamageMultiplier = float64(spell.baseDamageMultiplierAdditivePct) / 100.0
+	spell.baseDamageMultiplier = float64(100+spell.baseDamageMultiplierAdditivePct) / 100.0
 }
 
 func (spell *Spell) updateImpactDamageMultiplier() {
-	spell.impactDamageMultiplier = spell.damageMultiplier * (float64(spell.damageMultiplierAdditivePct+spell.impactDamageMultiplierAdditivePct-100) / 100.0)
+	spell.impactDamageMultiplier = spell.damageMultiplier * (float64(100+spell.damageMultiplierAdditivePct+spell.impactDamageMultiplierAdditivePct) / 100.0)
 }
 
 func (spell *Spell) updatePeriodicDamageMultiplier() {
-	spell.periodicDamageMultiplier = spell.damageMultiplier * (float64(spell.damageMultiplierAdditivePct+spell.periodicDamageMultiplierAdditivePct-100) / 100.0)
+	spell.periodicDamageMultiplier = spell.damageMultiplier * (float64(100+spell.damageMultiplierAdditivePct+spell.periodicDamageMultiplierAdditivePct) / 100.0)
```

Comment:
Just a different way to approach the formula really but this all avoids the `-100` offsets we had to do in some spell code

Reply:
Yea makes sense, better to do it for the devs then ask them to think about it.